ops/terraform/services/server/modules/bfd_server_asg/templates/fhir_server.tpl

```diff
@@ -18,8 +18,13 @@ aws ssm get-parameters-by-path \
     --region us-east-1 \
     --query 'Parameters' | jq 'map({(.Name|split("/")[5]): .Value})|add' > server_vars.json
 
-# build `data_server_ssl_client_certificates` from json values stored in SSM
-jq '{data_server_ssl_client_certificates: .data_server_ssl_client_certificates_json|fromjson}' server_vars.json > client_certificates.json
+# the previous ssm get-parameter will also pick the certs up due to the 'recursive' arg;.
+# we'll process the ".../client_certs/" path separately.
+aws ssm get-parameters-by-path \
+--path "/bfd/${env}/server/nonsensitive/client_certificates/" \
+--recursive --region us-east-1 \
+--query 'Parameters' | jq '.[] | {"alias": (.Name|split("/")[6]), "certificate": .Value}' \
+| jq -s '{ "data_server_ssl_client_certificates": . }' > client_certificates.json
```

Comment:
I might be missing something - should this be 

```
| jq -s . > client_certificates.json```
```

Ignore if `launch.yml` can process the json file regardless of the `data_server_ssl_client_certificates` object prefixing the array of certs

Reply:
Just noticed this was specifically changed in your recent commits, so the next question is if the other `fhir_server.tpl` needs to match this one

Reply:
the proverbial 'BFD tax'...there are two fhir_server.tpl files....only the one under the ASG umbrella is live....I inadvertently edited the wrong one at some point! TF NOT under the ASG tree will be going away!

As for the | jq -s . > client_certificates.json, I tried to minimize downstream changes, so the client_certificates.json is expecting a top-level element 'data_server_ssl_client_certificates'.

Reply:
gotcha :) I thought so re: old template but wanted to make sure!